From: Richard Hansen <rhansen@rhansen.org>
Date: Sat, 1 Aug 2020 16:22:38 -0400
Subject: Change the default server for `dslreports1` to `www.dslreports.com`

Before, it defaulted to `members.dyndns.org` which didn't make much
sense.

Forwarded: https://github.com/ddclient/ddclient/pull/264
Applied-Upstream: https://github.com/ddclient/ddclient/commit/25a636879f922234cb7ec8f1aa945f74d7cd2a38
---
 ddclient | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ddclient b/ddclient
index 71e69be..cfda556 100755
--- a/ddclient
+++ b/ddclient
@@ -569,6 +569,7 @@ my %services = (
 	'update'     => \&nic_dslreports1_update,
 	'examples'   => \&nic_dslreports1_examples,
 	'variables'  => merge(
+			  { 'server' => setv(T_FQDNP, 1, 0, 1, 'www.dslreports.com', undef) },
 			  { 'host' => setv(T_NUMBER,   1, 1, 1, 0, undef)       },
 			  $variables{'service-common-defaults'},
 		        ),
@@ -3195,7 +3196,6 @@ Configuration variables applicable to the 'dslreports1' protocol are:
 Example ${program}.conf file entries:
   ## single host update
   protocol=dslreports1,                                     \\
-  server=www.dslreports.com,                                \\
   login=my-dslreports-login,                                \\
   password=my-dslreports-password                           \\
   123456
